problemsets-mat223/modules/module1-exercises.tex

```diff
@@ -126,6 +126,84 @@
     		    \item Yes.
 		    \end{enumerate}
 		\end{solution}
+		
+		\prob Use set-builder notation to describe the following sets.
+		\begin{enumerate}
+			\item The set of vectors in $\R^{2}$ whose coordinates are rational numbers.
+
+			\item The set of vectors in $\R^{2}$ whose coordinates are irrational
+				numbers.
+
+			\item Let $P(\vec x) = \{\vec x, -\vec x\}$. The set $\{P(\vec e_{1}), P(
+				\vec e_{2})\}$.
+		\end{enumerate}
+
 		% Challenge (3 questions)
+		\prob % Propose: get students to understand quantifiers and set operations
+		% practice providing justification to their intuitive guesses
+		Which of the following statements is true about the set listed below? Justify your
+		answers.
+		\begin{enumerate}
+			\item $\mathcal{Y}$, the y-axis in $\R^{3}$.
+				\begin{enumerate}
+					\item $\mathcal{Y}$ is a finite set.
+
+					\item Let
+						$\mathcal{A} = \{\vec a \in \R^{3} : \vec a = \beta \vec v
+						\text{ for some }\vec v \in \mathcal{Y}, \beta \in \R \}$, then $\mathcal{A}
+						\subseteq \mathcal{Y}$.
+
+					\item For all vectors $\vec v \in \mathcal{Y}$, $\vec v \neq \vec 0$。
+
+					\item For some vectors $\vec v \in \mathcal{Y}$, $\vec v \neq \vec 0$.
+
+					\item For all vectors $\vec v \in \mathcal{Y}$, there exists a vector
+						$\vec x \in \mathcal{Y}$ such that $\vec x + \vec v = \vec e_{2}$.
+
+					\item There exists a vector $\vec x \in \mathcal{Y}$ such that for
+						all vectors $\vec v \in \mathcal{Y}$, $\vec x + \vec v = \vec e
+						_{2}$.
+				\end{enumerate}
+
+			\item $\mathcal{S}$, the set of vectors in $\R^{3}$ whose coordinates are $\pm
+				3$.
+				\begin{enumerate}
+					\item $\mathcal{S}$ is a finite set.
+
+					\item Let
+						$\mathcal{A} = \{\vec a \in \R^{3} : \vec a = \beta \vec v
+						\text{ for some }\vec v \in \mathcal{S}, \beta \in \R \}$, then $\mathcal{A}
+						\subseteq \mathcal{S}$.
+
+					\item For all vectors $\vec v \in \mathcal{S}$, $\vec v \neq \vec 0$.
+
+					\item For some vectors $\vec v \in \mathcal{S}$, $\vec v \neq \vec 0$.
+
+					\item For all vectors $\vec v \in \mathcal{S}$, there exists a vector
+						$\vec x \in \mathcal{S}$ such that $\vec x + \vec v = \vec 0$.
+
+					\item There exists a vector $\vec x \in \mathcal{S}$ such that for
+						all vectors $\vec v \in \mathcal{S}$, $\vec x + \vec v = \vec 0$.
+				\end{enumerate}
+		\end{enumerate}
+
+		\prob % Propose: Give students some practices on proving general statements about sets
+		Determine whether the following statement is correct or not. If it is, prove it.
+		Otherwise, give a counterexample.
+		\begin{enumerate}
+			\item For all sets $A \subseteq \R^{n}$, $A$ can be written as a set of linear
+				combinations of vectors in $\R^{n}$.
```

Comment:
I don't know what this means...

Reply:
I mean to ask: Can any set of vectors be written as a set of linear combination of vectors.

Reply:
So, would the answer "yes, because v=1v for all vectors. Therefore `X= { v : v = 1x for some x\in X}`" be what you are looking for?

Reply:
I agree that this isn't very intellectual stimulating... Will remove it!